cezo_fl/client.py

```diff
@@ -40,15 +89,15 @@ def __init__(
         self.local_update_seeds: list[int] = []
         self.local_update_dir_grads: list[torch.Tensor] = []
 
-    def random_gradient_estimator(self):
-        return self.grad_estimator
-
     def _get_train_batch_iterator(self):
         # NOTE: used only in init, will generate an infinite iterator from dataloader
         while True:
             for v in self.dataloader:
                 yield v
 
+    def random_gradient_estimator(self):
+        return self.grad_estimator
```

Comment:
add return type annotation. Also you can mimic what I did for device in the abstract class